pkg/apis/config/default.go

```diff
@@ -63,6 +66,7 @@ type Defaults struct {
 	DefaultCloudEventsSink            string
 	DefaultTaskRunWorkspaceBinding    string
 	DefaultMaxMatrixCombinationsCount int
+	ForbiddenEnvs                     []string
```

Comment:
Do we want to follow the same naming convention here? --`DefaultForbiddenEnvs`

Reply:
However, if you still feel that this should be named `DefaultForbiddenEnvs`, I can do it.

Reply:
Hi Prashant, I do prefer we follow the name convention, if we don't think this variable holds any default value, we may want to put it in other configmaps instead of the `default` one 🤔 ? But it is also acceptable if we say, by default, this variable is empty,  wdyt?